testing/functional/tests/cypress/integration/02-institution/01-login-institution-page.spec.ts

```diff
@@ -11,12 +11,20 @@ describe("Login Page", () => {
     cy.visit(url);
   });
 
-  it("Verify that user able to login with a valid username and valid password.", () => {
-    institutionCustomCommand.loginInstitution();
-  });
+  it(
+    "Verify that user able to login with a valid username and valid password.",
+    { retries: 4 },
```

Comment:
I noticed this configuration used in many places. Is there a way to do it globally? If not, can it be at least shared or is it something to be defined case by case?

Reply:
In institutionCustomCommand, a global method for login cases has already been created, and here I am making use of that method directly for login cases.

<img width="336" alt="image" src="https://user-images.githubusercontent.com/98920571/167524329-16e5a509-12bf-4b01-8b15-03a0bc94b489.png">

Reply:
I was talking specifically bout the `{ retries: 4 }`.